terraform/outputs.tf

```diff
@@ -0,0 +1,5 @@
+output "lambda_functions" {
+  value = [
+    module.download_grants_gov_db.lambda_function_name,
```

Comment:
Last bit of boilerplate. 

I found this output particularly useful for doing functional testing with LocalStack, where after running `tflocal apply`, I'd need to get the name of my Lambda function so I could pass it to `awslocal lambda invoke`, and this helped me avoid the need to go hunting for it.